consumer/src/main/resources/application.yml

```
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: notification-consumer
+  profiles:
+    active: local
+    include: event
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)